.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     '@typescript-eslint/no-unused-vars': ['warn', { argsIgnorePattern: '^_', varsIgnorePattern: '^_' }],
     // needed for inference from type guards
     '@typescript-eslint/no-var-requires': 0,
+    'brace-style': 0, // collides with prettier formatting
```

Comment:
Would be great to extract this into a separate commit

Also please squash the rest into a single `feat` commit that has a footer formatted like that

```
BREAKING CHANGE: add metadata to bip32 wallets
- split WalletRepository.updateMetadata into 2 methods
```

it will then bump the package version correctly as well as add notes to CHANGELOG.md

Reply:
done ✅

Reply:
The chore commit should be unscoped, as it applies to more than 1 (all in this case) package. Like this: `chore: disable brace style eslint rule for prettier co...`. Also it should be the 1st commit ,because the 2nd commit depends on it (you would get lint errors if you checked out that commit.